astropy/coordinates/sky_coordinate.py

```diff
@@ -1003,6 +1007,105 @@ def to_string(self, style='decimal', **kwargs):
 
         return coord_string
 
+    def to_table(self, table=None, column_conflicts='warn',
+                 metadata_conflicts='warn'):
+        """
+        Convert the data in this |SkyCoord| to a |QTable|.
+
+        Any attributes that have the same length as the |SkyCoord| will be
+        columns in the output. All other attributes will be recorded as
+        metadata.
+
+        If ``table`` is a |Table| whose length matches that of the |SkyCoord|
+        then the columns and metadata of the converted |SkyCoord| are merged
+        with it. In case of a column name or metadata conflict data from the
+        |SkyCoord| is preferred.
+
+        Parameters
+        ----------
+        table : |Table|, optional
+            A table to which the data from the |SkyCoord| is added.
+        column_conflicts : str ('warn'|'error'|'silent')
```

Comment:
https://numpydoc.readthedocs.io/en/latest/format.html
```suggestion
        column_conflicts : str {'warn','error','silent'}
```

Reply:
I believe I copied this line from here: https://github.com/astropy/astropy/blob/01ed9fe9eddd04aa97ccf475e5735231c8a88575/astropy/coordinates/sky_coordinate.py#L122
but I can't be entirely sure because there are also quite a few instances of this elsewhere. I can ensure the new docstring conforms to NumPy standards, and a separate pull request should address any examples in the code already present.